src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,71 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String RESULT = "result";
+    private static final String SUPPLY_OPERATOR = "supply";
+    private static final String BUY_OPERATOR = "buy";
+    private static final String COMMA = ",";
+    private static final int INDEX_OF_OPERATOR = 0;
+    private static final int INDEX_OF_NUMBER = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
This method is really big, it better to move report generation to the private method.